Create visualize_rent_prices_with_Streamlit.md

sixhobbits opened the pull request:
**Issue:** N/A

**Description:** 

We are enjoying Streamlit and we built a basic app and tutorial that extends the current data explorer example with some useful features that others can adapt.

We (https://ritza.co) are a startup that helps companies produce technical articles, tutorials, and documentation for developers. Let us know if you need any help with the Streamlit docs or more example content and we would love to work with you!

---

**Contribution License Agreement**

By submitting this pull request you agree that all contributions to this project are made under the Apache 2.0 license.

randyzwitch commented:
Hi @sixhobbits, thanks for your contribution. Before this can be merged, there are several issues:

1. The documentation does not build, according to our embedded ReadtheDocs check
2. There are several typos in the example
3. The links to the pictures are external to Streamlit-owned repositories. Please move the pictures into the streamlit /docs directory in the appropriate location.
4. For the section where you mention "Streamlit includes its own simpler visualization library", it is more correct to say that we provide convenience functions around Altair. Streamlit isn't building our own visualizations.

Note that by accepting this pull request, we are doing so as an open-source contribution, not as an agreement to pay you or your company for writing a tutorial.

Best,
Randy

sixhobbits committed:
add rent prices tutorial

sixhobbits committed:
Merge branch 'add-rent-price-tutorial' into develop

sixhobbits committed:
close backtick

sixhobbits committed:
clarify altair

sixhobbits commented:
Thanks for the response and feedback, Randy.

>     1. The documentation does not build, according to our embedded ReadtheDocs check

Fixed. It seems having the string \`$` in text was the issue in case that is interesting.

>     2. There are several typos in the example

We fixed several typos.

>     3. The links to the pictures are external to Streamlit-owned repositories. Please move the pictures into the streamlit /docs directory in the appropriate location.

Images moved into a local directory.

>     4. For the section where you mention "Streamlit includes its own simpler visualization library", it is more correct to say that we provide convenience functions around Altair. Streamlit isn't building our own visualizations.

Revised this sentence.
> Note that by accepting this pull request, we are doing so as an open-source contribution, not as an agreement to pay you or your company for writing a tutorial.

Of course :) We love Streamlit and sorry if the ad for our services was too commercial for the spirit of open source.

sixhobbits commented:
@randyzwitch - given that we addressed all of your feedback, let us know if this is of any use to you now.
